lang/en.yml

```diff
@@ -230,6 +230,9 @@ en:
     EMAILDETAILSTAB: 'Email Details'
     PLURALNAME: 'User Defined Form Email Recipients'
     SINGULARNAME: 'User Defined Form Email Recipient'
+    EMAILADDRESSINVALID: 'Email address has an invalid email address'
+    EMAILFROMINVALID: 'Email from has an invalid email address'
```

Comment:
Or maybe "[field name]" does not contain a valid email address (with quotes around the field name)